lib/generator.js

```diff
@@ -252,6 +255,10 @@ class Generator {
   async generateFromString(asyncapiString, parserOptions = {}) {
     if (!asyncapiString || typeof asyncapiString !== 'string') throw new Error('Parameter "asyncapiString" must be a non-empty string.');
 
+    if (Object.keys(this.mapBaseUrlToFolder).length > 0) {
```

Comment:
This `if` will be always true if someone use CLI, because in `cli.js` we have 

```js
const mapBaseUrlToFolder = { url: '', folder: process.cwd()};
```

Could you change it to:

```suggestion
    if (this.mapBaseUrlToFolder.url) {
```
? Then we have sure that someone pass url.